joint_trajectory_arm:
  ros__parameters:

    controller_name: "joint_trajectory_position_controller"
    wait_sec_between_publish: 4
    repeat: True
    
    goal_names: ["pos1", "pos2", "pos3", "pos4", "pos5"]
    pos1:
      positions: [1.57,   0.5,  0.5,  0.1]
    pos2:
      positions: [0.0,    0.0,  0.0, -1.0]
    pos3:
      positions: [-1.57, -0.3, -0.5, -0.1]
    pos4:
      positions: [0.0,    0.0,  0.0,  1.0]
    pos5:
      positions: [3.14,   0.0,  0.0,  0.0]

    joints:
      - shoulder_joint
      - upper_arm_joint
      - middle_arm_joint
      - fore_arm_joint

    check_starting_point: true
    starting_point_limits:
      shoulder_joint: [-3.14, 3.14]
      upper_arm_joint: [-1.5, 1.3]
      middle_arm_joint: [-1.5, 1.0]
      fore_arm_joint: [-1.5, 1.0]